lib/elastic_record/index/documents.rb

```diff
@@ -167,6 +173,10 @@ def scroll(scroll_id, scroll_keep_alive)
         end
       end
 
+      def clear_scroll(scroll_ids)
+        connection.json_delete('/_search/scroll', { scroll_id: scroll_ids })
+      end
```

Comment:
Belongs with other search methods. 

Name it delete_scroll

Reply:
All the methods around this are making these types of operations (e.g. making a request) through the connection. Where would you like this to go and why not here?

Are you just saying you'd like to see #scroll and #delete_scroll moved above build_scroll_enumerator?

Reply:
Ah sorry. I thought we had a Search module, like Manage. Never mind.